arch/arm64/Kconfig

```diff
@@ -29,6 +29,11 @@ config ARM_CUSTOM_INTERRUPT_CONTROLLER
 	  functions are mapped to the SoC interrupt control interface, which is
 	  implemented at the SoC level.
 
+config QEMU_KVM
+	bool
+	help
+	  This option enables KVM acceleration
```

Comment:
`arch/arm64/Kconfig` is really not the right place for this config.

This should belong in `boards/Kconfig` alongside other QEMU-related Kconfigs:
https://github.com/zephyrproject-rtos/zephyr/blob/c11b7852d15c2920303cda813ddd1642c433fb18/boards/Kconfig#L58